collector/src/compare.rs

```diff
@@ -0,0 +1,110 @@
+use database::{Connection, Lookup};
+use itertools::Itertools;
+use tabled::{Table, Tabled};
+
+const DEFAULT_SIGNIFICANCE_THRESHOLD: f64 = 0.002;
+
+/// Compare 2 artifacts and print the result.
+pub async fn compare_artifacts(mut conn: Box<dyn Connection>, base: String, modified: String) {
+    let index = database::Index::load(&mut *conn).await;
+    let sids = index
+        .compile_statistic_descriptions()
+        .map(|(_, sid)| sid)
+        .collect::<Vec<_>>();
+
+    let base_id_number = conn
+        .artifact_by_name(&base)
+        .await
+        .expect("Cannot find specified artifact")
+        .lookup(&index)
+        .unwrap();
+    let modified_id_number = conn
+        .artifact_by_name(&modified)
+        .await
+        .expect("Cannot find specified artifact")
+        .lookup(&index)
+        .unwrap();
+
+    let pstats = conn
+        .get_pstats(&sids, &[Some(base_id_number), Some(modified_id_number)])
+        .await;
+    let tuple_pstats = pstats
+        .into_iter()
+        .map(|row| row.into_iter().collect_tuple::<(_, _)>().unwrap())
```

Comment:
Could you just implement this by hand? The code uses `unwrap` anyway, so it would crash if there weren't exactly two elements. Introducing a new dependency just for this feels like overkill.